From 06b4d6f40cbcb8be2930421e026563193725c875 Mon Sep 17 00:00:00 2001
From: jon <jon.hefling@lairdtech.com>
Date: Wed, 11 Dec 2013 10:23:34 -0600
Subject: [PATCH] Remove RTC driver from kernel.

The RTC hardware is not available, so no need for kernel module.
Bug 4523.
---
 arch/arm/configs/wb45n_defconfig |    2 --
 1 file changed, 2 deletions(-)

diff --git a/arch/arm/configs/wb45n_defconfig b/arch/arm/configs/wb45n_defconfig
index 73e9fc7..1dac885 100644
--- a/arch/arm/configs/wb45n_defconfig
+++ b/arch/arm/configs/wb45n_defconfig
@@ -238,8 +238,6 @@ CONFIG_LEDS_TRIGGER_CPU=y
 CONFIG_LEDS_TRIGGER_GPIO=y
 CONFIG_LEDS_TRIGGER_DEFAULT_ON=y
 CONFIG_LEDS_TRIGGER_TRANSIENT=y
-CONFIG_RTC_CLASS=y
-CONFIG_RTC_DRV_AT91SAM9=y
 CONFIG_DMADEVICES=y
 CONFIG_NET_DMA=y
 # CONFIG_IOMMU_SUPPORT is not set
-- 
1.7.9.5

